geomstats/_backend/pytorch/__init__.py

```diff
@@ -28,13 +31,15 @@
     float64,
     floor,
     fmod as mod,
-    ger as outer,
-    gt as greater,
+    outer,
+    greater,
+    hstack,
     int32,
     int64,
     isnan,
     log,
-    lt as less,
+    logical_or,
```

Comment:
`torch.logical_and` seems to exist as well: https://pytorch.org/docs/1.8.1/generated/torch.logical_and.html?highlight=logical_and#torch.logical_and

Reply:
> `torch.logical_and` seems to exist as well: https://pytorch.org/docs/1.8.1/generated/torch.logical_and.html?highlight=logical_and#torch.logical_and

@nguigs yup. But somehwere in the codebase, `logical_and` is used for non-torch tensor type and so we have to keep what is currently. 

```
def logical_and(x, y):
    if torch.is_tensor(x):
        return x & y
    return x and y

```

Reply:
Actually. Quite a bit of code can be removed if we don't use these backend related functions in heterogeneous way  (mixing numpy arrays, torch tensors, lists and passing to backend functions)